source/Calamari.Common/Features/Scripting/WindowsPowerShell/Bootstrap.ps1

```diff
@@ -292,6 +292,40 @@ function Write-Verbose
 	}
 }
 
+<#
+.SYNOPSIS
+Writes text to stderr when running in a regular console window,
+to the host''s error stream otherwise.
+
+.DESCRIPTION
+Writing to true stderr allows you to write a well-behaved CLI
+as a PS script that can be invoked from a batch file, for instance.
+
+Note that PS by default sends ALL its streams to *stdout* when invoked from
+cmd.exe.
+
+This function acts similarly to Write-Host in that it simply calls
+.ToString() on its input; to get the default output format, invoke
+it via a pipeline and precede with Out-String.
+
+taken from https://stackoverflow.com/a/15669365/19177210
```

Comment:
I lifted this from a stack overflow post - is that ok?

Reply:
Yeah I'm not sure - don't want to get done for copywrite but there isn't many ways you can actually write this...

I'm tempted to just write it as a warning instead in powershell and not check the error level in the test.

Reply:
I've removed this.